atl-checker/src/edg.rs

```diff
@@ -53,6 +53,16 @@ pub fn distributed_certain_zero<
     let assignment = manager_broker
         .receive_result()
         .expect("Error receiving final assigment on termination");
+
+    let mut assignment_table = HashMap::<V, VertexAssignment>::new();
+    for _ in 0..worker_count {
+        let assignments = manager_broker
+            .receive_assignment()
+            .expect("Error receiving the assignment table");
+
+        assignment_table.extend(assignments);
```

Comment:
Extending the assignment table overrides values, but certain assignments needs to have priority, i.e. if one worker found a certain assignment, then that must be the one put into the table. Unknown < undecided < {true, false}.